staff/joseluis-juste/JS/proyect2.js

```diff
@@ -0,0 +1,238 @@
+Array.prototype.showAll = function()
```

Comment:
Why are you writing your code like this?

```
Array.prototype.showAll = function()

	{
		this.forEach(function(x){
 			console.log("ID: " + x.id + ", Origen: " + x.from + ", Destino: " + x.to + ", Coste: " + x.cost + ", Escala: " + x.scale);
		});
	};
```
instead of 

```
Array.prototype.showAll = function(){
		this.forEach(function(x){
 			console.log("ID: " + x.id + ", Origen: " + x.from + ", Destino: " + x.to + ", Coste: " + x.cost + ", Escala: " + x.scale);
		});
	};
```

Reply:
Fixed in commit JS SyntaxFixes.